pkg/devspace/config/loader/save.go

```diff
@@ -73,6 +73,17 @@ func (l *configLoader) RestoreVars(config *latest.Config) (*latest.Config, error
 	return clonedConfig, nil
 }
 
+var keysNoRestoration = []string{"images", "deployments", "dev"}
```

Comment:
rename to ignoreConfigKeys

Reply:
Fixed in ae4cc7473c71012c13d04ff7e0852409b14d34af